requirements-11.0.txt

```
@@ -0,0 +1,5 @@
+https://raw.githubusercontent.com/OCA/OCB/11.0/requirements.txt
```

I think this is required:

```suggestion
-r https://raw.githubusercontent.com/OCA/OCB/11.0/requirements.txt
```